event_session_registration_multi_qty/tests/test_event_session_registration_multi_qty.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017-19 Tecnativa - David Vidal
```

Comment:
You can use a smaller filename instead of test_event_session_registration_multi_qty

Reply:
Well, the name is test_ + module name, and there's no problem in that scheme IMO.

Reply:
I know, but the module name is too long and there are other modules that use a shorter name in this test files

Reply:
Sorry, I don't see a problem with it, and it's a convention to us. :)